src/schemas/shared/getEntityByIdParamSchema.ts

```diff
@@ -0,0 +1,17 @@
+import { t } from "elysia";
+
+export const GetEntityByIdQueryParamSchema = t.Object(
+  {
+    id: t.String({
```

Comment:
probably should put the xmple directly in the id, examples is buggy i just used it first because i wasnt sure.